### PR TITLE
BXMSDOC-4943-master: Installing and configuring DM/PAM 7.6 in an EAP clustered environment

### PR DESCRIPTION
**Pull request by csherrar**
Changes:
- Correct smart router file name version
- Add HTTPS configuration and keystore properties
- Modify procedure
- Update NFS sections and requirements
- Modify nfs server procedure

### CODE REVIEW

File: doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+= Configuring an NFS server
+
+You must deploy and configure an NFS server to provide the shared file system necessary for a {CENTRAL} clustered environment and each node client must have access to the shared file system.
+
+You must use NFS version 4.
+
+.Procedure
+
+. Configure a server to export NFS version 4 shares. For instructions about exporting NFS shares on Red Hat Enterprise Linux, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/exporting-nfs-shares_managing-file-systems[Exporting NFS shares]. For more information about creating the NFS server, see https://access.redhat.com/solutions/1355233[_How to configure NFS in RHEL 7_].
```
Comment (emmurphy1) [inline doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc]: It's best to just link to the TOC in books that you have no control over, then direct the user to the appropriate section in words. You never know when the author might rearrange the chapters.

File: doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ These steps describe a basic cluster configuration. For more complex configurati
 
 * {DATAGRID} {DATAGRID_VERSION_LONG} is installed as described in <<clustering-datagrid-proc_{context}>>.
 * {EAP} is installed on each node of the cluster as described in <<clustering-download-eap-bc-proc>>.
-* An NFS server with a mounted partition is available.
+* An NFS server with a mounted partition is available as described in <<nfs-server-configure-proc>>.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc]: Quite right, done

File: doc-content/enterprise-only/installation/clustering-smartrouter-install-proc.adoc [hunk 3]
```diff
@@ -26,8 +26,9 @@ Load balancing requests for Smart router cluster must be managed externally, usi
 * *Version: {PRODUCT_VERSION}*
 . Download *{PRODUCT} {PRODUCT_VERSION_LONG} Add-Ons*.
 . Extract the downloaded `{PRODUCT_FILE}-add-ons.zip` file to a temporary directory. The `{PRODUCT_INIT}-{PRODUCT_VERSION}-smart-router.jar` file is in the extracted `{PRODUCT_FILE}-add-ons` directory.
-. Copy the `{PRODUCT_INIT}-7.0-smart-router.jar` file to location where you will run the file.
-. From the temporary directory, enter the following command to start SmartRouter:
+. Copy the `{PRODUCT_INIT}-{PRODUCT_VERSION}-smart-router.jar` file to location where you will run the file.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-smartrouter-install-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-smartrouter-install-proc.adoc [hunk 4]
```diff
@@ -61,20 +62,32 @@ The router is capable of providing an aggregate sort, however the data returned
...
++
+----
+org.kie.server.router.tls.keystore = <KEYSTORE_PATH>
+org.kie.server.router.tls.keystore.password = <KEYSTORE_PWD>
+org.kie.server.router.tls.keystore.keyalias = <KEYSTORE_ALIAS>
+org.kie.server.router.tls.port = <HTTPS_PORT>
+-jar kie-server-router-proxy-7.10.0-SNAPSHOT.jar
+----
++
+`org.kie.server.router.tls.port` is an optional property to configure the HTTPS port. The default HTTPS port value is `9443`.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-smartrouter-install-proc.adoc]: Done

File: doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
+[id='nfs-server-configure-proc']
+
+= Configuring an NFS server
+
+You must deploy and configure an NFS server to provide the shared file system necessary for a {CENTRAL} clustered environment and each node client must have access to the shared file system.
+
+You must use NFS version 4.
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc]: Done, Rephrased

File: doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc]: Good point, done

File: doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+. Configure a server to export NFS version 4 shares. For instructions about exporting NFS shares on Red Hat Enterprise Linux, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/exporting-nfs-shares_managing-file-systems[Exporting NFS shares]. For more information about creating the NFS server, see https://access.redhat.com/solutions/1355233[_How to configure NFS in RHEL 7_].
+
+. On the server, create an `/opt/kie/data` share with the `rw,sync,no_root_squash` options by adding the following line to the `/etc/exports` file:
++
+[subs="attributes,verbatim,macros"]
+----
+/opt/kie/data *(rw,sync,no_root_squash)
+----
++
+Where `/opt/kie/data` is the shared folder, `*` are the IP addresses allowed to connect to the NFS server, and `(rw,sync,no_root_squash)` are the minimum options required for NFS. For example:
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc]: Done

File: doc-content/enterprise-only/installation/clustering-smartrouter-install-proc.adoc [hunk 7]
```diff
@@ -1,12 +1,12 @@
 [id='clustering-smartrouter-install-proc']
 = Clustering {KIE_SERVERS} with Smart Router
 
-You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can both route requests to individual {KIE_SERVERS} and aggregate data from different {KIE_SERVERS}. Through aliases, Smart Router is a proxy to {KIE_SERVERS}. Smart Router performs the following tasks:
+You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can route requests to individual {KIE_SERVERS} and aggregate data from different {KIE_SERVERS}. Smart Router is a proxy to {KIE_SERVERS} by using aliases. Smart Router performs the following tasks:
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/installation/clustering-smartrouter-install-proc.adoc]: Done

**Pull request merged**